### ISSUE TITLE
Radeon Software 21.4.1+ AVC/HEVC Failed to start recording

### ISSUE THREAD

**Issue by Neoony** [Xaymar/OBS-AMD-Advanced-Media-Framework#418]
OBS not working with AVC/HEVC encoding with latest AMD drivers (21.4.1 or higher)

I created issue at OBS repo, not gonna recreate the whole issue at this repo, but check here: 
https://github.com/obsproject/obs-studio/issues/4689

Hoping for more recognition. Did not realize AMD support is separate git repo.

**Comment by Lithium64** [comment]
Same issue here, I had to go back to driver 21.3.2 to be able to use the AMD encoder.

**Comment by Neoony** [comment]
Ok, so I went to discord to ask and Jessica | tielqt suggested to DDU and install new drivers again.
Although I tried that with drivers before 21.6.2

But this time with 21.6.2 it helped.

DDU in safe mode and restart
then installed newest chipset drivers 2.17.25.506 (X370)
then installed newest AMD drivers 21.6.2
Restarted

And now I get no error with OBS anymore

so for me with Radeon VII, its resolved.
